src/acrcssc/azext_acrcssc/helper/_taskoperations.py

```diff
@@ -160,7 +174,8 @@ def delete_continuous_patch_v1(cmd, registry, dryrun):
         delete_oci_artifact_continuous_patch(cmd, registry, dryrun)
 
     if not cssc_tasks_exists:
-        logger.warning(f"{ERROR_MESSAGE_WORKFLOW_TASKS_DOES_NOT_EXIST}")        
+        logger.warning(f"{ERROR_MESSAGE_WORKFLOW_TASKS_DOES_NOT_EXIST}")
+
```

Comment:
Shall we add a log for dry run?

Reply:
'delete' code path should not have a 'dry-run' scenario, removed in latest commits